data/categories.txt

```diff
@@ -818,6 +820,7 @@ nl:3Wisselkantoor|3geld wisselen|wisselen|geld
 fi:4Rahanvaihto
 fr:3Bureau de change|3change|3devises|argent
 de:3Geldwechselstelle|Wechselstube|Geld|Geldumtausch
+hi:विनिमय कार्यालय
```

Comment:
Will someone type this long phrase exactly as you wrote it here? Does it make sense to add a number before it to display as a hint? The same is related to other cases below.